ek9000App/src/devEK9000.cpp

```diff
@@ -945,7 +946,7 @@ void ek9000ConfigureTerminal(const iocshArgBuf* args) {
 
 void ek9000Stat(const iocshArgBuf* args) {
 	const char* ek9k = args[0].sval;
-	if (!ek9k || !g_pDeviceMgr) {
```

Comment:
Was `g_pDeviceMgr` just not used? I guess that's point (3) in your PR description

Reply:
That is correct, g_pDeviceMgr was left over